README.md

```diff
@@ -90,6 +90,21 @@ make devnet-clean
 
 We are open to contributions, but don't currently have a formal process for contributors. If you are interested, browse through [open issues](https://github.com/omni-network/omni/issues), read our [code of conduct](./CODE_OF_CONDUCT.md), or chat with the team.
 
+Follow these steps to set up a functional development environment:
+
+1. Install Docker Desktop.
+2. Run `make install-pre-commit`.
```

Comment:
did you need to run `make install-go-tools` as well, or was that not necessary?

Reply:
Ah, I installed all Go tools via Vim plugin command, that's why it wasn't necessary for me, but you're right — in a general case, we need it, so I'll add it.